docs/src/whatsnew/latest.rst

```diff
@@ -37,6 +39,26 @@ This document explains the changes made to Iris for this release
 ✨ Features
 ===========
 
+#. `@bjlittle`_, `@pp-mo`_ and `@trexfeathers`_ added support for unstructured
+   meshes, as described be UGRID. This involved adding a data model (:pull:`3968`,
+   :pull:`4014`, :pull:`4036`, :pull:`4053`) and API (:pull:`4063`, :pull:`4064`),
+   and supporting representation (:pull:`4033`, :pull:`4054`) and loading
+   (:pull:`4058`, :pull:`4074`) of data on meshes.
```

Comment:
I'm impressed you've managed to pick out the PR's from the maelstrom! Not sure what your criteria were, but looking through [all `mesh-data-model` PR's](https://github.com/SciTools/iris/pulls?page=1&q=is%3Apr+base%3Amesh-data-model) (with some human knowledge thrown in), I'm wondering whether you should also include:

* #4057 
* #4049 
* #4037 
* #4027 - definitely this one
* #4014
* #3968

What do you think?

Reply:
These do look like they're worth mentioning. Just wondering for #4027, while this has a large footprint in terms of lines changed, it seems like it is more of a logistical change as a mergeback than an actual piece of work like, say, #4015, which it contains. I guess the benefit of #4027 is that it contains references to multiple PRs, though linking major pieces of work like #4015 seems like it would refer closer to the actual work.

Reply:
Yeah I can see the confusion here. The `ng-vat-mesh-api` branch was used as a development branch for multiple developers to collaborate on a single addition. So the end output of `ng-vat-mesh-api` is the intended focus.

_As opposed to our regular use of branches as feature branches, where each PR is more 'publicly consumable'. None of this is helped by the heavy over-use of the term 'merge-back' to refer to branch merges for many **different** purposes._

Reply:
I've made some changes to the listed PRs on the principle that chosen PRs should represent a significant change to functionality (minor PRs, PRs focused on testing and internal changes are ignored).